crates/zkvm/entrypoint/src/syscalls/bigint_uma.rs

```diff
@@ -0,0 +1,38 @@
+use super::syscall_u256x2048_mul;
+
+/// The number of limbs in a "uint256".
+const N: usize = 8;
+#[allow(unused_variables)]
```

Comment:
perhaps this existed in where u copied from, but there should be a line between 4 and 5.